conan_tests/external_tools/bare_mingw_test.py

```diff
@@ -52,12 +52,12 @@ def diamond_mingw_test(self):
                 self.client.run("install %s -s compiler=gcc -s compiler.libcxx=libstdc++ " 
                                 "-s compiler.version=4.9 --build=missing" % path_dot())
                 self.client.run("build .")
-
-        command = os.sep.join([".", "bin", "say_hello"])
-        self.client.run_command(command)
-        self.assertEqual(['Hola Hello4', 'Hola Hello3', 'Hola Hello1', 'Hola Hello0',
-                          'Hola Hello2', 'Hola Hello0'],
-                         str(self.client.out).splitlines()[-6:])
+                
+                command = os.sep.join([".", "bin", "say_hello"])
```

Comment:
why? The execution of the binary really shouldn't depend on having mingw or removing bash

Reply:
It's because the execution of the binary was failing because `libstdc++6.dll` was not found in the path.